### PR TITLE
content/agent/docker: add new environment variables for APM Docker

### PR DESCRIPTION
**Pull request by gbbr**
Being released this week in agent 6.9.0

### CODE REVIEW

File: content/agent/docker/apm.md [hunk 1]
```diff
@@ -40,20 +40,24 @@ docker run -d -v /var/run/docker.sock:/var/run/docker.sock:ro \
...
+| `DD_APM_DD_URL`            | Datadog API endpoint where traces are sent. For Datadog EU region set `DD_APM_DD_URL` to `https://trace.agent.datadoghq.eu` |
+| `DD_APM_RECEIVER_PORT`     | Port that the Datadog Agent's trace receiver listen on. Default value is `8126`.                                            |
+| `DD_APM_NON_LOCAL_TRAFFIC` | Allow non-local traffic when [tracing from other containers](#tracing-from-other-containers).                               |
+| `DD_APM_IGNORE_RESOURCE`   | A comma-separated list of endpoints and resources that the Agent should ignore (i.e. health checks endpoints).              |
+| `DD_APM_ANALYZED_SPANS`    | Configure the spans to analyze for transactions.                                                                            |
+| `DD_APM_ENV`               | Sets the default [environment][3] for your traces.                                                                          |
+| `DD_APM_MAX_EPS`           | Sets the maximum events per second.                                                                                         |
```
Comment (l0k0ms) [inline content/agent/docker/apm.md]: @gbbr when we say events here, do we mean traces sent? or APM events for trace search.

Comment (gbbr) [reply content/agent/docker/apm.md]: APM events, corrected that.

File: content/agent/docker/apm.md [hunk 2]
```diff
@@ -40,20 +40,24 @@ docker run -d -v /var/run/docker.sock:/var/run/docker.sock:ro \
...
-| `DD_LOG_LEVEL`             | Set the logging level. (`trace`/`debug`/`info`/`warn`/`error`/`critical`/`off`)                                |
-| `DD_APM_ANALYZED_SPANS`    | Configure the spans to analyze for transactions.                                                               |
+| Environment variable       | Description                                                                                                                 |
+| ------                     | ------                                                                                                                      |
+| `DD_API_KEY`               | [Datadog API Key][2]                                                                                                        |
+| `DD_PROXY_HTTPS`           | Set up the URL for the proxy to use.                                                                                        |
+| `DD_HOSTNAME`              | Set manually the Agent hostname.                                                                                            |
```
Comment (cswatt) [inline content/agent/docker/apm.md]:
```suggestion
| `DD_HOSTNAME`              | Set the Agent hostname manually.                                                                                            |
```

**Comment by gbbr** [comment]
Agent 6.9.0 was released yesterday. This is good to merge now.

**Pull request merged**